servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -171,21 +171,18 @@ def query_jobs(body, **kwargs):
     headers = kwargs.get('auth_headers')
     query = QueryJobsRequest.from_dict(body)
     query_page_size = query.page_size or _DEFAULT_PAGE_SIZE
-    # Request more than query.page_size from cromwell since subworkflows will get filtered out
-    page_size = query_page_size * 2
     total_results = get_total_results(query, auth, headers)
-
     results = []
     offset = page_tokens.decode_offset(query.page_token) or 0
-    page = page_from_offset(offset, page_size)
-    last_page = get_last_page(total_results, page_size)
+    page = page_from_offset(offset, query_page_size)
+    last_page = get_last_page(total_results, query_page_size)
 
     while len(results) < query_page_size and page <= last_page:
```

Comment:
We can get rid of this while loop now that we no longer have to filter out subworkflows and continually make additional requests to get a full page of results. Instead, we can make a request to cromwell for a specific page.

Reply:
thanks – will do